packages/webpack-cli/lib/groups/resolveOutput.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+const parseDirectory = (metaData) => {
+    return {
+        // filename: DEFAULT_FILENAME,
```

Comment:
What is this?

Reply:
not sure, old code 😄 

Lemme try getting rid of this